### PR TITLE
Ccj

### PR DESCRIPTION
**Pull request by charucjoshi**
Removed redundant code and files.

### CODE REVIEW

**Review by sharath2106 — CHANGES_REQUESTED** [review]
Have asked for some changes which is more about structuring the implementation for better maintainability and readability.

File: github_push.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import streamlit as st
+from urllib.parse import urlparse
+from github import Github
+
+# GitHub credentials
+def push(access_token, yaml_content, repository_link):
```
[the remark this answers is not in the document]
Comment (sharath2106) [reply github_push.py]: I would keep it push_to_github because yaml is already a parameter.

[the remark this answers is not in the document]
Comment (charucjoshi) [reply github_push.py]: ok... changed the method name to push_to_github which was previously the name of the push to github button. Button's name changed to push_to_github_button.

**Comment by sharath2106** [comment]
Looks good to me.

**Pull request merged**